scripts/lib/cluster.sh

```diff
@@ -7,6 +7,14 @@ function load_cluster_details() {
   K8S_VERSION=$(echo "$DESCRIBE_CLUSTER_OP" | jq .cluster.version -r)
 }
 
+function load_deveks_cluster_details() {
+
+  echo "loading cluster details $CLUSTER_NAME"
+  PROVIDER_ID=$(kubectl get nodes --kubeconfig $KUBE_CONFIG_PATH -ojson | jq -r '.items[0].spec.providerID')
+  INSTANCE_ID=${PROVIDER_ID##*/}
```

Comment:
Why do we have `##*/` here? Perhaps for a comment?

Reply:
It's a way to fetch the instance id from the string provider id. It removes all the characters upto the last '/'
aws:///us-west-2a/i-xxxxxxxxxxxxx -> i-xxxxxxxxxxxxx